src/CMakeLists.txt

```diff
@@ -113,6 +119,10 @@ if(APPLE AND X86_64)
 set(compiler_flags_release ${compiler_flags_global})
 elseif(X86_64)
 set(compiler_flags_release ${compiler_flags_global})
+elseif(PPC64LE)
+set(compiler_flags_release ${compiler_flags_global} -mcpu=power8)
```

Comment:
POWER8 is the first to support Little Endian. This is the lowest common value for all CPUs that supports LE for example POWER8, POWER9 and POWER10

ref: https://www.ibm.com/docs/en/openxl-c-and-cpp-aix/17.1.0?topic=options-mcpu